numpy/ma/core.py

```diff
@@ -6066,10 +6075,9 @@ def __reduce__(self):
 masked_array = MaskedArray
 
 
-def array(data, dtype=None, copy=False, order=False,
-          mask=nomask, fill_value=None,
-          keep_mask=True, hard_mask=False, shrink=True, subok=True, ndmin=0,
-          ):
+def array(data, dtype=None, copy=False, order=None,
```

Comment:
Actually, I'm just changing the default value of ```order``` here (for some reason it was defaulted to ```False``` before).  I don't think I have to put it at the end in this case?